go.mod

```diff
@@ -473,3 +483,5 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+replace github.com/google/go-containerregistry => github.com/google/go-containerregistry v0.14.1-0.20230409045903-ed5c185df419
```

Comment:
Why upgrade this package and then use a replace here for an older version?

Reply:
because trivy and cosign deps are for different versions and the newer version breaks.

Reply:
Maybe it's better to use top of main for cosign?

Reply:
Actually I think we're ok if everything compiles successfully and the tests pass